packages/admin-ui/src/utils/validation.ts

```diff
@@ -26,9 +27,22 @@ export function validateMinLength(
   return null;
 }
 
+export function validateMiaxLength(
```

Comment:
Is this function used somewhere?

```suggestion
export function validateMaxLength(
```